apps/judicial-system/web/src/components/FeatureProvider/FeatureProvider.tsx

```diff
@@ -6,10 +6,7 @@ interface FeatureProvider {
   features: Feature[]
 }
 
-const availableFeatures: Feature[] = [
-  Feature.INDICTMENTS,
-  Feature.INDICTMENT_ROUTE,
-]
+const availableFeatures = Object.values(Feature)
```

Comment:
The only difference here is now the availableFeatures array will always include Feature.NONE, while with the previous code it only had INDICTMENTS and INDICTMENT_ROUTE. It doesn't seem to affect the functionality at all though, but we could also remove it from the availableFeatures list if we want.

Reply:
I think this is fine.